otel-extensions/src/main/java/org/hypertrace/agent/otel/extensions/config/EnvironmentConfig.java

```diff
@@ -130,6 +133,14 @@ private static void applyPropagationFormat(AgentConfig.Builder builder) {
     if (traceReportingType != null) {
       builder.setTraceReporterType(TraceReporterType.valueOf(traceReportingType));
     }
+    String metricReporterAddress = getProperty(REPORTING_METRIC_ENDPOINT);
+    if (metricReporterAddress != null) {
```

Comment:
The `EnvironmentConfig` test should be expanded to also test these new properties

Reply:
done